coordinator/src/services/translation/request/dispense/release.ts

```diff
@@ -39,9 +39,15 @@ export async function createNominatedReleaseRequest(
   const hl7Id = new hl7V3.GlobalIdentifier(uuid.v4())
   const timestamp = convertMomentToHl7V3DateTime(moment.utc())
   const hl7Release = new hl7V3.NominatedPrescriptionReleaseRequest(hl7Id, timestamp)
+  const practitonerRoleTelecom = practitionerRole.telecom[0].value
```

Comment:
Typo in practitioner

Reply:
Could pass just the telecom value in rather than the entire PractitionerRole. Might make testing simpler?

Reply:
Is there any validation to ensure that a telecom is present? Might null pointer here if not